.circleci/config.yml

```diff
@@ -0,0 +1,60 @@
+version: 2.1
+executors:
+  node-executor:
+    docker:
+      - image: circleci/node:10.15
+    working_directory: ~/keystone
+
+commands:
+  set-npm-auth:
+    description: "Set NPM authentication"
+    steps:
+      - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/repo/.npmrc
+
+  install-deps:
+    description: "Install build dependencies"
+    steps:
+      - restore_cache:
+          keys:
+            - v1-dependencies-build-{{ checksum "yarn.lock" }}
+      - run: yarn install --forzen-lockfile
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-build-{{ checksum "yarn.lcok" }}
+
+jobs:
+  build:
+     executor: node-executor
+     steps:
+       - checkout
+       - install-deps
+       - run: yarn build
+       - persist_to_workspace:
+          root: ~/keystone
+          paths:
+            - ./*
+  publish:
+    executor: node-executor
+    working_directory: ~/keystone
+    steps:
+      - attach_workspace:
+          at: .
+      - set-npm-auth
+      - run: npm publish
+
+workflows:
+  version: 2.1
+  build-and-publish:
+    jobs:
+      - build
+      - publish:
+          context: twreporter
```

Comment:
Q: Currently, how do we use context ? Is it a shared environment variables across projects ?

Reply:
Yes, this is across projects. It's very hard to revoke the same tokens that are defined specifically in each project.

Reply:
BTW, we already use the context for the definition of the credentials of Kubernetes clusters.